packages/roosterjs-editor-plugins/lib/plugins/CustomReplace/CustomReplace.ts

```diff
@@ -11,7 +11,7 @@ const makeReplacement = (
     sourceString: string,
     replacementHTML: string,
     matchSourceCaseSensitive: boolean,
-    shouldReplace?: (replacement: CustomReplacement, content: string) => boolean
+    shouldReplace?: (replacement: CustomReplacement, content: string, contentTag: string) => boolean
```

Comment:
This is a public parameter, add it as a required parameter will cause a breaking change. Please, check if add it is really necessary, if is, add as optional.

Reply:
contentTag?: string

Reply:
Does not that leave the parameter as undefined instead of optional?

Reply:
https://www.typescripttutorial.net/typescript-tutorial/typescript-optional-parameters/

That means the parameter is optional. Caller can decide whether pass it or not. And when it is not passed, its value will be undefined. So in your function you need to check if it is undefined before use.

Reply:
I did not know typescrpt had that notation. I was confusing it with the ternary operator because they use the same notation. Good to know now.